tutorials/nservicebus-101/lesson-1/tutorial.md

```diff
@@ -3,7 +3,9 @@ title: "NServiceBus 101 Lesson 1: Getting started"
 reviewed: 2017-01-26
 ---
 
-In the next 10 minutes, you will learn how to set up a new development machine for NServiceBus and create your very first messaging endpoint.
+partial: nsb101-intro-paragraph
+
+In this first lesson, which should take no longer than 10-15 minutes, you will learn how to set up a new development machine for NServiceBus and create your very first messaging endpoint.
```

Comment:
> which should take no longer than 10-15 minutes

The "no longer" part conflicts with the range of 10-15 minutes. Suggest either:

> which should take no longer than 15 minutes

or (my preference):

> which should take 10-15 minutes